src/qt/guiutil.cpp

```diff
@@ -1007,4 +1009,15 @@ void ShowModalDialogAsynchronously(QDialog* dialog)
     dialog->show();
 }
 
+void bringToFrontWorkaroundForWayland(QWidget* w)
```

Comment:
Any specific reason to not inline this code?

Reply:
Just to have it separately, also in case we need to touch it again.

Reply:
This makes diff bigger, you need to touch the header. Also, on Wayland, the `activateWindow()` and `raise()` can be unneeded.

Reply:
Ok, I'll add a conditional there then.